app/cdap/components/WranglerGrid/SelectColumnPanel/utils.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import {
+  IHeaderNamesList,
+  IMultipleSelectedFunctionDetail,
+} from 'components/WranglerGrid/SelectColumnPanel/types';
+import { MULTI_SELECTION_COLUMN } from 'components/WranglerGrid/SelectColumnPanel/constants';
+
+/**
+ * @param  {string[]} transformationDataType
```

Comment:
don't include type definitions in the js doc comments if its a typescript file. The type definitions in the functions don't need to be repeated

Reply:
updated the JS Docs
removed type definitions from JS Docs